src/styles/blocks.scss

```diff
@@ -0,0 +1,126 @@
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  font-family: $body-font;
+  width: $card-width;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid $border-color;
+  padding: 32px 16px 16px;
+  background-color: $card-color;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__text {
+    display: flex;
+    flex-direction: column;
+    width: 166px;
```

Comment:
Setting fixed width is not the best idea, width of the text should be determined by the place it has available.